CMakeLists.txt

```diff
@@ -15,8 +15,7 @@ set(VERSION_MINOR "6")
 set(VERSION_PATCH "1")
 set(VERSION_ABI 1)
 
-set(DEFLECT_DESCRIPTION
-  "The client library to write applications for DisplayCluster")
+set(DEFLECT_DESCRIPTION "The C++ library to stream pixels to DisplayCluster")
```

Comment:
It's actually not bound to DisplayCluster anymore. You can stream and receive pixels just with Deflect.